### ISSUE TITLE
[Feature] - Initialise state for sorting functionality

### ISSUE THREAD

**Issue by Dorfieeee** [jdwilkin4/fcc-forum-clone#68]
### What feature would you like to see?

This issue follows on adding sorting functionality discussed in https://github.com/jdwilkin4/fcc-forum-clone/discussions/50.
We have created buttons and attached click events on them so far as you can see in the Fig.1 below.
Another step is to make them do stuff.

### Task 1
For this task, you need to initialise two global variables, sortingBy and sortingOrder.
You need to assign them with some meaningful value representing the default state. ( it has not been set by the user yet )

### Task 2
By clicking on any of the buttons, we will need to decide whether the posts should be sorted in ascending, descending, or default order. That could be achieved by storing some value in `sortingOrder`. But what kind of value? It could be set of strings such as "ascending", "descending" , "default" or set of integers such as 0, 1, and 2. This is for you to decide and discuss with others. 
In other words, the sorting order can have 3 states: default, ascending, and descending.
You need to come up with a solution, how to easily keep track between these 3 states using the `sortingOrder`. 
There are no restrictions.

<figure id="fig1">
  <img width="422" alt="image" src="https://user-images.githubusercontent.com/65303812/219781746-258b43aa-9028-4bba-b690-8038473e7cb4.png">
  <figcaption>Fig.1 - Sorting buttons</figcaption>
</figure>
